core/src/main/java/io/questdb/cairo/DefaultCairoConfiguration.java

```diff
@@ -184,7 +184,8 @@ public int getCreateTableModelPoolCapacity() {
 
     @Override
     public long getDataAppendPageSize() {
-        return getFilesFacade().getMapPageSize();
+        return 2 * 1024 * 1024;
+//        return getFilesFacade().getMapPageSize();
```

Comment:
Looks like a leftover. Also, we have `DefaultTestCairoConfiguration` used in all tests, so it's better to override this method there and fine-tune it for tests.

Reply:
I append performance earlier and it is not really necessary to pre-allocate 16MB even for embedded apps. You'd have to be requiring crazy ingestion speed to need that. Also, `DefaultTestCairoConfiguration` is not used _everywhere_.

Leftover removed here https://github.com/questdb/questdb/pull/2938/commits/1affeee8acf1396b85765e973d3a5c3ff73bd9ee

Reply:
@bluestreak01 I'd move the overridden value to `DefaultTestCairoConfiguration`. Otherwise, this change impacts embedded API users.

Reply:
> `DefaultTestCairoConfiguration` is not used _everywhere_.

Well, the idea was to use it in all tests. Anyhow, I'm fine with changing the value to 2MB for embedded API users. I'd even lower down the default value in `PropServerConfiguration`. Marking this one as resolved.